code/modules/clothing/suits/jobs.dm

```diff
@@ -56,6 +56,23 @@
 		rad = 0
 	)
 
+/obj/item/clothing/suit/storage/cargo_jacket/old
+	name = "black guild technician jacket"
+	desc = "Stylish jacket lined with pockets. It seems to have a little protection from physical harm. This one is done in dark black color."
+	icon_state = "cargo_jacket_old"
+	item_state = "cargo_jacket_old"
+	blood_overlay_type = "coat"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	style_coverage = COVERS_TORSO|COVERS_UPPER_ARMS
+	armor = list(
+		melee = 2,
+		bullet = 2,
+		energy = 2,
+		bomb = 0,
+		bio = 0,
+		rad = 0
+	)
```

Comment:
```suggestion
```

No need for these since it's a cosmetic difference and will inherit the values from `/obj/item/clothing/suit/storage/cargo_jacket`.